### PR TITLE
Add -P --prefix option

### PR DESCRIPTION
**Pull request by felfert**
While figuring out why my custom tag pattern did not work, I stumbled over
the prefix parameter of the TagFilter class. This turned out to be the ideal
functionality to be use with [this](https://github.com/felfert/catlock) repository where the tags start with 'v'.

This PR adds a new option -P --prefix for specifying a prefix.
The next PR will fix the actual bug where the custom pattern is not used when
creating TagFilter object.

### CODE REVIEW

**Comment by felfert** [comment]
Wait a second with this one. I'll add a test case for that.

**Comment by weakcamel** [comment]
> Wait a second with this one. I'll add a test case for that.

You're reading my mind :-) Thank you!

**Comment by felfert** [comment]
Will resubmit my other PR once this one is merged.
This way I can reuse test-code from this PR in the other one.

File: gcg/entrypoint.py [hunk 1]
```diff
@@ -143,6 +143,13 @@ def parse_args(argv):
         help="Verbosity level. Specify twice for debug logs as well.",
         action="count", default=0
     )
+    parser.add_argument(
+        '-P', '--prefix',
```
Comment (weakcamel) [inline gcg/entrypoint.py]: How would you feel about making it --tag-prefix?
It's a little bit longer yet also more descriptive and specific (tag prefix? filename prefix? )

**Comment by weakcamel** [comment]
Awesome work, thank you!

I vaguely recall that the tag filter functionality seemed mostly complete yet I didn't manage to test it properly before I had to switch to some other project in the past.

If you're ready to go with this one - let me know and I'll merge it in.

File: gcg/entrypoint.py (hunk 1 above)
Comment (felfert) [reply gcg/entrypoint.py]: Shure. Just a moment ...

**Comment by weakcamel** [comment]
Off we go!

**Comment by weakcamel** [comment]
Dev version should soon be available on https://test.pypi.org/project/gcg/

**Pull request merged**